app/data/test_navigation.json

```diff
@@ -0,0 +1,667 @@
+        {
+            "blocks": [
+                {
+                    "type": "questionnaire",
+                    "display": {
+                        "properties": {}
+                    },
+                    "id": "household-composition",
+                    "sections": [
+                        {
+                            "display": {
+                                "properties": {}
+                            },
+                            "id": "section",
+                            "questions": [
+                                {
+                                    "description": "",
+                                    "display": {
+                                        "properties": {}
+                                    },
+                                    "id": "household-composition-question",
+                                    "title": "Who usually lives at the property?",
+                                    "description": "",
+                                    "type": "RepeatingAnswer",
+                                    "validation": [],
+                                    "answers": [
+                                        {
+                                            "alias": "person_full_name",
+                                            "display": {},
+                                            "id": "household-full-name",
+                                            "label": "Full Name",
+                                            "mandatory": true,
+                                            "options": [],
+                                            "q_code": "1",
+                                            "type": "TextField",
+                                            "validation": {},
+                                            "repeats": true
+                                        }
+                                    ]
+                                }
+                            ],
+                            "title": "Household Details",
+                            "validation": []
+                        }
+                    ],
+                    "title": "Household Details",
+                    "validation": []
+                }
+            ],
+            "display": {
+                "properties": {}
+            },
+            "id": "multiple-questions-group",
+            "title": "Household Details",
+            "validation": [],
+            "completed_when": "household-composition"
+        },
+        {
+            "blocks": [
+                {
+                    "type": "questionnaire",
+                    "id": "repeating-block-1",
+                    "title": "Block 2",
+                    "sections": [
+                        {
+                            "title": "{{answers.person_full_name[group_instance]}}",
+                            "description": "",
+                            "id": "91631df0-4356-4e9f-a9d9-ce8b08d26eb3",
```

Comment:
I think it makes sense for the id above to say as it is however these being UUIDs don't make it clearer it obfuscates it